manifests/p/ParallelsInternationalGmbH/ParallelsClient-64bit/18.0.22546/ParallelsInternationalGmbH.ParallelsClient-64bit.yaml

```
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: ParallelsInternationalGmbH.ParallelsClient-64bit

```

```suggestion
PackageIdentifier: Parallels.Parallels
```
**Please make sure to update the folder names and file names using GitHub Desktop from https://desktop.github.com on Windows.**